docs/commands/got.md

```diff
@@ -0,0 +1,28 @@
+## Command got ##
+
+Display the current state of GOT table of the running process.
+
+The `got` command optionally takes function names and filters 
+the output displaying only the matching functions. 
+```
+gef➤ got
+```
+
+![gef-got](https://i.imgur.com/NHceezH.png)
```

Comment:
you're going to want to kill me, but could you regen these with the RIGHT_ARROW?

Reply:
oh no ahah
ok ok, I'm also into this kind of details, will do that!